src/CMakeLists.txt

```diff
@@ -80,10 +90,14 @@ ADD_EXECUTABLE(play_sequence
 )
 TARGET_LINK_LIBRARIES(play_sequence aps2)
 
-ADD_EXECUTABLE(test_comms
-	./util/test_comms.cpp
+ADD_EXECUTABLE(run_tests
+	../test/run_tests.cpp
+	../test/test_Sanity.cpp
+	../test/test_datagram.cpp
+	./lib/APS2Datagram.cpp
+	../test/test_comms.cpp
 )
-TARGET_LINK_LIBRARIES(test_comms aps2)
+TARGET_LINK_LIBRARIES(run_tests aps2 pthread)
```

Comment:
Sorry, I take that back. `pthread` should work on both `gcc` and `clang`, so we can keep it here.